app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestViewModel.kt

```diff
@@ -399,54 +367,85 @@ class MeetingRequestViewModel(
    * the contact is too far away
    */
   fun meetingDistanceCancellation() {
```

Comment:
Here what you are doing is that you update the inbox and then you modify it. It should be the other way around

Reply:
I don't think it is an issue here since the idea is to get the latest version of the users in messaging radius and the the users we contacted before making the decision to cancel the meeting request or not

Reply:
Okay you get the current inbox and then modify it, but after that you should update inbox once again to store the changes. Since update inbox is called at multiple times it doesn't causes errors here, but if it happens you will need to come back to add the update inbox.